app/controllers/admin/volunteers.py

```diff
@@ -51,30 +49,35 @@ def trackVolunteersPage(eventID):
         if volunteer.user not in volunteerUser:
             eventVolunteerData.append(volunteer)
             volunteerUser.append(volunteer.user)
+    eventWaitlistData = [volunteer for volunteer in eventVolunteerData if volunteer.rsvpWaitlist]
     eventLengthInHours = getEventLengthInHours(event.timeStart, event.timeEnd, event.startDate)
 
     recurringEventID = event.recurringId # query Event Table to get recurringId using Event ID.
     recurringEventStartDate = event.startDate
     recurringVolunteers = getPreviousRecurringEventData(recurringEventID)
 
+    currentRsvpAmount = getCurrentRsvpAmount(g.current_term)
+
     return render_template("/events/trackVolunteers.html",
-        eventData = eventData,
-        eventVolunteerData = eventVolunteerData,
-        eventParticipants = eventParticipants,
-        eventLength = eventLengthInHours,
-        event = event,
-        recurringEventID = recurringEventID,
-        recurringEventStartDate = recurringEventStartDate,
-        recurringVolunteers = recurringVolunteers,
-        bannedUsers = bannedUsers,
-        trainedParticipantsList = trainedParticipantsList)
+                            eventData = eventData,
+                            eventVolunteerData = eventVolunteerData,
+                            eventParticipants = eventParticipants,
+                            eventLength = eventLengthInHours,
+                            event = event,
+                            recurringEventID = recurringEventID,
+                            recurringEventStartDate = recurringEventStartDate,
+                            recurringVolunteers = recurringVolunteers,
+                            bannedUsers = bannedUsers,
+                            trainedParticipantsList = trainedParticipantsList,
+                            eventWaitlistData = eventWaitlistData,
+                            currentRsvpAmount = currentRsvpAmount)
 
 @admin_bp.route('/eventsList/<eventID>/track_volunteers', methods=['POST'])
 def updateVolunteerTable(eventID):
     try:
         event = Event.get_by_id(eventID)
     except DoesNotExist as e:
-        print(f"No event found for {eventID}")
+        print(f"No event found for {eventID}", e)
```

Comment:
I wouldn't output the exception. We know it is a DoesNotExist exception and it's only going to be thrown by one line so it is obvious. Outputting the other message is sufficient so we know which one wasn't found